azdev/utilities/command.py

```diff
@@ -21,10 +21,9 @@ def call(command, **kwargs):
     :param kwargs: Any kwargs supported by subprocess.Popen
     :returns: (int) process exit code.
     """
-    from azdev.utilities import IS_WINDOWS
     return subprocess.call(
-        command.split(),
-        shell=IS_WINDOWS,
+        command,
+        shell=True,
```

Comment:
why previously it's IS_WINDOWS? I suppose it is for local running azdev right? @haroldrandom do you by chance know it? Just don't want to break existing experience since azdev is important to our daily work.

Reply:
The author didn't write any natural language description of this piece of tricky code. We cannot make progress if we are always trapped in history.

Reply:
Use `shlex.split` is working. No need to change this.

Reply:
> 
> 
> Use `shlex.split` is also working. No need to change this.

Not working in Powershell. And the test result file location is wrong.

Reply:
About the original design why shell=True is missing, I am not sure either.
But in this case, I think using `shlex.split(command, posix=not IS_WINDOWS)` would be more goal-oriented, could leave the history design aside.

Reply:
Still not working. It selects all tests.

Reply:
According the [doc](https://docs.python.org/3/library/subprocess.html#popen-constructor), 
> On Windows, if args is a sequence, it will be converted to a string in a manner described in [Converting an argument sequence to a string on Windows](https://docs.python.org/3/library/subprocess.html#converting-argument-sequence). This is because the underlying CreateProcess() operates on strings.